doc/man1/openssl-pkcs12.pod.in

```diff
@@ -58,6 +58,7 @@ B<openssl> B<pkcs12>
 {- $OpenSSL::safe::opt_r_synopsis -}
 {- $OpenSSL::safe::opt_engine_synopsis -}
 {- $OpenSSL::safe::opt_provider_synopsis -}
+[B<-legacy>]
```

Comment:
Maybe move this up before the "common" options. And similarly below with the legacy option description.